helm/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "indexd.fullname" . }}-test-connection"
+  labels:
+    {{- include "indexd.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "indexd.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
I think we also need to point this to the `/_status` endpoint. We aren't really using `helm test` right now, though.

Reply:
Yeah didn't get this to work with istio yet, and also we don't do any tests on this as of now :(

Reply:
ok np